post_start/script/set_default_graf_dash.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/sh
 
+# Grafana connection configuration
```

Comment:
nit: comments should always be full sentences, including punctuation.

Reply:
Changed to "# Set your Grafana instance's connection configuration.".